Bottom: 9545934a45e4fb1c76a911297a383a953487ef9d
Top:    563d3a14f5f5450f582cc4e221a86ea1f4145b9b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-06 21:25:23 +0100

Refresh of give-the-hids-control-over-att

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 55f8c74..9be7297 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -309,8 +309,8 @@ ghid_port_key_release_cb (GtkWidget * drawing_area, GdkEventKey * kev,
 
   notify_crosshair_change (false);
   AdjustAttachedObjects ();
-  ghid_invalidate_all ();
   notify_crosshair_change (true);
+  ghid_invalidate_all ();
   ghid_screen_update ();
   g_idle_add (ghid_idle_cb, NULL);
   return FALSE;
